schema-registry/serdes/src/main/java/com/hortonworks/registries/schemaregistry/serdes/avro/AbstractAvroSnapshotDeserializer.java

```diff
@@ -138,19 +151,13 @@ protected Object buildDeserializedObject(InputStream payloadInputStream,
                 // generate UTF-8 string object from the received bytes.
                 deserializedObj = new String(IOUtils.toByteArray(payloadInputStream), AvroUtils.UTF_8);
             } else {
+                //This is deprecated but kept to keep protocol.
```

Comment:
I am fine with removing this in serializer also breaking compatibility with earlier version. It is better to avoid this compatibility baggage in these early initial versions. We will maintain backward compatibility from the upcoming release.